docs/Reporting.md

```diff
@@ -1,58 +1,66 @@
 # FCA: Reporting
 
-TODO: add intro
+FCA is pre-configured to generate detailed mochawesome JSON reports following the completion of [sanity tests](./Execution.md). This functionality not only provides users with immediate test results but also enables sending these reports to AWS or any other URL endpoint for further processing and analysis. Reporting needs will vary on a case-by-case basis and users have the flexibility to utilize the generated mochawesome report JSON in various ways, including transforming the JSON data into an HTML report.
 
 ## Table of Contents
 
-- [Supported Ways of Retrieving Reports](#supported-ways-of-retrieving-reports)
-- [Supported Report Parameters](#supported-report-parameters)
+- [Using the Existing Reporting Logic](#using-the-existing-reporting-logic)
+- [Creating Your Own Reporting Logic](#creating-your-own-reporting-logic)
 
-## Supported ways of retrieving reports
-
-EXISTING CONTENTS(needs revision):
-```
-- From UI
-
-  - After executing suite from the UI in FCA, the mocha json will upload to S3.
-  - Download the `json report` from S3 and convert it to `html report`
-    - `arn:aws:s3:::your-s3-Bucket-name`
-  - Steps to convert into `html report`
-    1.  Move the `json report` to root folder
-    2.  Run the command `yarn marge filename.json -f report -o /destination_path`
-    3.  Check the `html report` on the destination path.
-```
+## Using the Existing Reporting Logic
 
 ### Background
 
-TODO: add details
+The default reporting mechanism in FCA is designed to streamline your test analysis process. Post-execution of sanity tests, the app generates detailed mochawesome JSON reports. These reports are crucial for reviewing test outcomes. The `northBoundSchemaValidationAndReportGeneration()` function in [Test_Runner.js](../src/Test_Runner.js) orchestrates this process. Additionally, for users leveraging `IntentReader`, the [runTestHandler](../src/pubsub/handlers/RunTestHandler.js) class can be utilized to execute a sanity test and obtain a reports.
+
+#### Standalone Mode
+
+Furthermore, FCA supports `standalone` report generation. Standalone reporting posts to a different endpoint which allows for more diverse report management and usage.
```

Comment:
I think what I was trying to explain here is that standalone reports allow two things:

1 - You can pass a jobId which will name the report
2- You can post the report json to a different endpoint so that you have the ability to have two different endpoints that can handle the report json in two entirely different ways if you would like.

Reply:
So on 2, what is nice about a second endpoint? Thinking it is so you can configure another lambda/other API to do additional post report generation data manipulation. I think that is what you are getting at, but if we could say it a little more descriptively I think that would help. Maybe a use case for what endpoint a and b do and why two are helpful would suffice.

Reply:
Yes that is what I am trying to get it. Will add a little more content to be more descriptive for the reader.

Reply:
For example, you can configure Endpoint A to perform the initial report generation, while Endpoint B, a lambda function or API, can be set up to perform additional post-report generation data manipulation. This dual-endpoint approach allows you to efficiently customize and extend your reporting workflow, ensuring that your data is processed and analyzed exactly as needed.